web/src/pages/AboutPage/AboutPage.tsx

```diff
@@ -1,19 +1,44 @@
 import { Link, routes } from '@redwoodjs/router'
 import { MetaTags } from '@redwoodjs/web'
 
+import Person from './Person'
+
 const AboutPage = () => {
   return (
     <>
       <MetaTags title="About" description="About page" />
-
-      <h1>AboutPage</h1>
-      <p>
-        Find me in <code>./web/src/pages/AboutPage/AboutPage.tsx</code>
-      </p>
-      <p>
-        My default route is named <code>about</code>, link to me with `
-        <Link to={routes.about()}>About</Link>`
-      </p>
+      <div className="grid grid-cols-2 grid-rows-3 gap-4 pb-4">
+        <Person
+          name="Adriana"
+          quote="えーどりあな"
+          story="Insert story about yourself here"
```

Comment:
Perhaps for now we can use empty strings here because no story might look better than "Insert story about yourself here". On the other hand, we aren't really "live" so it also doesn't matter that much.